javagen/src/main/java/com/azure/autorest/mapper/ServiceClientMapper.java

```diff
@@ -139,10 +139,10 @@ public ServiceClient map(CodeModel codeModel) {
             }
         }
         serviceClientProperties.add(new ServiceClientProperty("The HTTP pipeline to send requests through.",
-                ClassType.HttpPipeline, "pipeline", true, null));
```

Comment:
If you do intend to use "pipeline", I can accommodate it in fluentgen, as it is not exposed to customer so not counted as breaking changes (though better not to change if no very good reason).

Reply:
it's the inconsistency between the naming in the client and the builder that drove me to make the change - but feel like it's probably less painful to just avoid this change